<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>235</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>332</width>
    <height>235</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>332</width>
    <height>235</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>332</width>
    <height>235</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/icons.qrc">
    <normaloff>:/icons/icon.ico</normaloff>:/icons/icon.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Main</string>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>17</y>
       <width>260</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>277</x>
       <y>17</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Constants (if need)</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>16</y>
       <width>46</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>X=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>16</y>
       <width>45</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>83</x>
       <y>16</y>
       <width>46</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Y=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>103</x>
       <y>16</y>
       <width>45</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>177</x>
       <y>16</y>
       <width>45</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>157</x>
       <y>16</y>
       <width>46</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Z=</string>
     </property>
    </widget>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>lineEdit_3</zorder>
    <zorder>lineEdit_4</zorder>
    <zorder>label_3</zorder>
    <zorder>lineEdit_2</zorder>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>8</x>
       <y>16</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>38</x>
       <y>16</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>!</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>311</width>
      <height>127</height>
     </rect>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>16</y>
       <width>291</width>
       <height>101</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="lineWrapMode">
      <enum>QPlainTextEdit::NoWrap</enum>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections/>
</ui>
